igor-web/src/test/groovy/com/netflix/spinnaker/igor/docker/DockerRegistryCacheV2KeysMigrationSpec.groovy

```diff
@@ -59,7 +59,10 @@ class DockerRegistryCacheV2KeysMigrationSpec extends Specification {
             keyFactory, igorConfigurationProperties, Schedulers.immediate())
     }
 
-    def tearDown() {
+    void cleanup() {
```

Comment:
Why was this renamed?  I'm not familiar with Igor's testing.

Reply:
it should have been cleanup not tearDown. The tearDown never got called and leak the the redis in memory. After several runs of "gradlew clean build", my laptop memory was full of redis because of these.

Reply:
Ah, good to know, thanks!